decode_test.go

```diff
@@ -2570,8 +2578,8 @@ func TestDecoder_LiteralWithNewLine(t *testing.T) {
 
 func TestDecoder_TabCharacterAtRight(t *testing.T) {
 	yml := `
-- a: [2 , 2] 			
```

Comment:
Please do not delete the tab character, as it is intentionally inserted.

Reply:
Ah, good to know. That's my formatter removing trailing spaces I guess.